client/app/components/dialog-editor-boxes/dialog-editor-boxes.component.js

```diff
@@ -0,0 +1,97 @@
+/* eslint camelcase: ["error", {properties: "never"}] */
+
+(function() {
+  'use strict';
+
+  angular.module('app.components')
+    .component('dialogEditorBoxes', {
+      controller: function(DialogEditor, DialogEditorModal, lodash) {
+        /**
+         * Load component data
+         */
+        this.$onInit = function() {
```

Comment:
Should be vm (see https://github.com/ManageIQ/manageiq-ui-service/blob/master/client/app/components/explorer/pagination/pagination.component.js#L19) 

all subsequent instances of `this` should be vm